src/common/clearAll.js

```diff
@@ -0,0 +1,7 @@
+const clearAll = () => {
+  /* eslint-disable */
+    localStorage.removeItem('toDolist');
+    location.reload();
+}
+
+export { clearAll }
```

Comment:
- [ ] As I mentioned in the previous comment(s), Kindly make sure you fix all your `linters` errors instead of disabling them as you did for best practices and clean code.

Reply:
Changed the same